src/Components/Aspire.Microsoft.Azure.Cosmos/AspireMicrosoftAzureCosmosDBExtensions.cs

```diff
@@ -83,6 +83,8 @@ public static class AspireMicrosoftAzureCosmosDBExtensions
         configureSettings?.Invoke(settings);
 
         var clientOptions = new CosmosClientOptions();
+        clientOptions.ApplicationName = CosmosConstants.CosmosApplicationName;
```

Comment:
Just an FYI that users can (and probably will) overwrite this value. We show them how to do this in the README:

https://github.com/dotnet/aspire/blob/87c39e0422f288e2c5a5ac0349ea33f514bc7d8a/src/Components/Aspire.Microsoft.Azure.Cosmos/README.md?plain=1#L105-L109

Reply:
Thanks! This was a good catch. In order to address this, I have prefixed the application name with whatever value the users provide. This will make sure that even if the user doesn't set the application name, it will have a default value when it reaches to out service.